src/app/public/modules/datepicker/datepicker-calendar-inner.component.ts

```diff
@@ -347,10 +347,10 @@ export class SkyDatepickerCalendarInnerComponent implements OnInit, OnChanges {
     }
   }
 
-  public toggleModeCalendar(event: Event, direction: number) {
+  public toggleModeCalendar(event: Event) {
     event.preventDefault();
     event.stopPropagation();
-    this.toggleMode(direction);
+    this.toggleMode(1);
```

Comment:
Do you mind providing some background for this change?

Reply:
This was another template error that was being caught by the build. `toggleModeCalendar` was asking for two arguments, but all the callers in the template were never using it. `direction` was always `undefined`.